modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -9537,7 +9476,7 @@
                     {
                       "title": "Resource not found",
                       "status": "404",
-                      "detail": "Could not find an Organization with code: 083",
+                      "detail": "Could not retrieve Power of Attorney with that registration number: invalid",
```

Comment:
this is interesting. two notes
1. "with that registration number" I think should be adjusted to "with registration number"
2. Invalid doesn't look good here. we could use one from the mocking like 083